projects/ngrx.io/content/guide/signals/signal-store/entity-management.md

```diff
@@ -336,6 +340,36 @@ const Store = signalStore(
 
 Try to avoid multiple entity types in one store. It is better to have multiple stores, each with a single entity type.
 
+---
+
+To avoid repetitive code, it's recommended to use the `entityMeta` function when defining a custom entity configuration:
+
+```typescript
+const todoMeta = entityMeta({
```

Comment:
Should we add some docs about what each property does, here and/or with JSDocs?
We can do this later.

Reply:
Entity meta properties are used and described in previous examples on this page.

I agree regarding JSDoc 👍 I plan to add it for all public APIs.